cmake/modules/FindMaya.cmake

```diff
@@ -331,6 +332,30 @@ if (MAYA_OGSDEVICES_LIBRARY)
 endif()
 message(STATUS "Using Maya Light API Version ${MAYA_LIGHTAPI_VERSION}")
 
+set(MAYA_DEFAULT_MATERIAL_API 0)
+# Grep for the new API in OpenMayaRender. Might be quicker than asking MayaPy.
+find_file(MAYA_OPENMAYARENDER_LIBRARY
+        "${MAYA_DSO_PREFIX}OpenMayaRender${MAYA_DSO_SUFFIX}"
+    HINTS
+        "${MAYA_LIBRARY_DIR}"
+        "${MAYA_LOCATION}"
+    PATH_SUFFIXES
+        lib/
+        bin/
+    DOC
+        "Maya's ${MAYA_LIB} library path"
+    # NO_CMAKE_SYSTEM_PATH needed to avoid conflicts between
+    # Maya's Foundation library and OSX's framework.
+    NO_CMAKE_SYSTEM_PATH
+)
+if (MAYA_OPENMAYARENDER_LIBRARY)
+    file(STRINGS ${MAYA_OPENMAYARENDER_LIBRARY} HAS_DEFAULT_MATERIAL_API REGEX "setDefaultMaterialHandling")
```

Comment:
This is a good idea and I'm wondering if we should use it more generally. Does it take much time to run the search?

Reply:
It was quite fast, but it will be even faster (and more precise) to look directly in the header file. Fixed that.